admin_ui/components/Spinner.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const Spinner = ({size, className}) => {
```

Comment:
Realised that we don't have a nice big (can be small!) reusable spinner!

Ended up coding this. I suggest we work with it a bit to see if we like it (i.e. does not annoy the hell out of us) and reuse in other places as well:

![image](https://user-images.githubusercontent.com/14941382/69012535-94ca3d80-097f-11ea-9565-c4dd940178eb.png)